cmd/captplanet/run.go

```diff
@@ -46,6 +49,7 @@ type Satellite struct {
 	StatDB      statdb.Config
 	BwAgreement bwagreement.Config
 	Web         satelliteweb.Config
+	DatabaseURL string `help:"the master database connection string" default:"sqlite3://$CONFDIR/master.db"`
```

Comment:
Let's call this just `Database`, I think the `URL` part isn't exactly correct and should be obvious from context.